keyboards/planck/keymaps/rjhilgefort/keymap.c

```diff
@@ -0,0 +1,167 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
+#include "action_layer.h"
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
+  BACKLIT,
+  EXT_PLV
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+
+#define CTRL_ESC MT(MOD_LCTL, KC_ESC)
+#define HYPER LCTL(LALT(QK_LGUI))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+* ,-------------------------------------------------------------------------------------------------------.
+* | Tab    |    Q   |    W   |    E   |    R   |   T   |   Y   |   U  |    I   |    O   |   P    | Bksp   |
+* |--------+--------+--------+--------+--------+-------+-------+------+--------+--------+--------+--------|
+* | Ct/Esc |    A   |    S   |    D   |    F   |   G   |   H   |   J  |    K   |    L   |   ;    |  "     |
+* |--------+--------+--------+--------+--------+-------+-------+------+--------+--------+--------+--------|
+* | Shift  |    Z   |    X   |    C   |    V   |   B   |   N   |   M  |    ,   |    .   |   /    | Enter  |
+* |--------+--------+--------+--------+--------+-------+-------+------+--------+--------+--------+--------|
+* |        |  Ctrl  | Raise  |   Alt  |   GUI  |     Space     | Lowr | Hyper  |        |        | Raise  |
+* `-------------------------------------------------------------------------------------------------------'
+*/
+[_QWERTY] = {
```

Comment:
Could you update these so that they're using the LAYOUT format from the default keymap? 

https://github.com/qmk/qmk_firmware/blob/9986c3d0b7f10cc07a3c38d8f46b4a157479e33a/keyboards/planck/keymaps/default/keymap.c#L57-L62